### DISCUSSION TITLE
LF4 MyOpenMath

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#355] [General]
Reviewer: @AbbyANoble 

I don't know if we want to keep this order (or even keep all of them), but I tried to group the ones that were similar in what they are asking.

Questions 1 - 3: Calculate the slope (given points) and then determine the slope of a line that is parallel.
Questions 4 - 9: When given an equation, find the slope of a line parallel/identify parallel lines from equation
Questions 10 - 21: Find equation of a line that is parallel to given equation and a point (mix of slope-intercept and point-slope).
Questions 22 - 24: Calculate the slope (given points) and then determine the slope of a line that is perpendicular.
Questions 25 - 29: Find equation of a line that is perpendicular to given equation and a point (mix of slope-intercept and point-slope).
Questions 30 - 34: Determine parallel/perpendicular lines from slope
Questions 35 - 42: Test whether lines are parallel or perpendicular (mix)
Questions 43 - 47: Find equations of parallel and perpendicular lines (mix)

**Comment by AbbyANoble** [comment]
I like 4-6, but they have a part where students explain their answer that will have to be manually graded. I'm not a big fan of those in homework because I use unlimited attempts and it may turn in to an unlimited number of things to keep up with and check. We should probably mark those so people know they require extra grading.

**Reply by tdegeorge** [reply]
What do you mean by marking them?

**Reply by AbbyANoble** [reply]
Just making a note wherever we post these for people so they know. Or deleting them. Either way!

**Comment by AbbyANoble** [comment]
Question 14 asks for **the** equation rather than **an** equation. Accepts point-slope and slope-intercept.

**Reply by AbbyANoble** [reply]
Same in number 17. Asks for **the** equation of **a** line.

**Reply by AbbyANoble** [reply]
And 18. Maybe I'm extra pedantic about this? But this is one of those tiny hills I die on in algebra. hahaha

**Reply by tdegeorge** [reply]
But if we edit the question, doesn't it reset the stats or something? Do we want to edit it?

I don't have a preference but I can see where you are coming from!

**Reply by AbbyANoble** [reply]
Yeah. It probably doesn't matter that much, but we could edit. Or just use the ones that have the language we like.

**Comment by AbbyANoble** [comment]
The assignment has a lot of good questions, but is very long.

**Reply by tdegeorge** [reply]
Yeah...I just didn't know what people liked (like how the questions were being asked).
